packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol

```
@@ -338,6 +381,65 @@ contract VerifyOPCM is Script {
         return success;
     }
 
+    /// @notice Verifies that the immutable variables in the OPCM contract match expected values.
+    /// @param _opcm The OPCM contract to verify immutable variables for.
+    /// @return True if all immutable variables are verified, false otherwise.
+    function _verifyOpcmImmutableVariables(IOPContractsManager _opcm) internal returns (bool) {
+        console.log("  Verifying OPCM immutable variables...");
+
+        bool success = true;
+
+        // Get all OPCM getters and iterate over them
+        string[] memory allGetters = _getOpcmGetters();
+
+        for (uint256 i = 0; i < allGetters.length; i++) {
+            string memory functionName = allGetters[i];
+            string memory verificationMethod = expectedGetters[functionName];
+
+            // Skip getters that don't need env var verification
+            if (
+                bytes(verificationMethod).length == 0
```

Would this line allow us to leave out an expectedGetters entry?

---

Theoretically it could, fixed it on the commit 7ae57fc7ed8f5ab44ca9b5dde1ec297e6264ad70 by failing fast